website/app/styles/components/do-dont.scss

```diff
@@ -26,17 +28,17 @@
 }
 
 .doc-do-dont__content {
-  padding: 12px;
+  padding: 16px;
   border-style: solid;
-  border-width: 1px;
+  border-width: 3px 1px 1px 1px;
```

Comment:
according to the `Banner` design specs, the `border-left` width in that case is `4px` (you can [see it in action here](https://hds-website-git-new-docs-website-content-block-654209-hashicorp.vercel.app/testing/components/banner/), in [this PR](https://github.com/hashicorp/design-system/pull/791/))

Do you think they should be aligned, or because this applies to the top side, `4px` would be too strong visually?

Reply:
Good catch, there's some discrepancy in the Figma specs. I'll see if we can clean it up, but will update to 4px.